docs/Themes.md

```diff
@@ -0,0 +1,333 @@
+# Themes
+
+Themes are used to customise the colors of the card. github-readme-testaustime ships with a whopping 60 themes to choose from.
+
+<details>
+    <summary>default</summary>
+  
+![image](https://github.com/HEPOSHEIKKI/github-readme-testaustime/assets/51783206/6a726abe-5508-412a-954d-e1823dbc630f)
```

Comment:
Oh yeah, these image urls are a bit weird, not sure how they will work when merged and fork is deleted.